web/frontend/src/components/slam-2d-render.vue

```diff
@@ -192,6 +307,38 @@ onUnmounted(() => {
   disposeScene();
 });
 
+// see if we can just do props.destVector and not the subcomponents
+watch(() => [props.destVector, props.destExists], async () => {
+  if (props.destVector && props.destExists) {
+    // update name from marker to destinationMarker
+    const marker = scene.getObjectByName('Marker') ?? await makeMarker(destMarkerUrl, 'Marker', 0.1);
+    marker.position.set(props.destVector.x + 1.2, 0, props.destVector.z +2.64 );
+  }
+  if (!props.destExists) {
+    const marker = scene.getObjectByName('Marker');
+    if (marker !== undefined) {
+      scene.remove(marker);
+    }
+  }
+});
+
+watch(() => props.axesVisible, () => {
+  const ax1 = scene.getObjectByName('AxesPos');
```

Comment:
[opt] Totally optional, but consider if it would be worth storing these in a tuple/list/dict called 'ax'.

Reply:
will do